src/waddle/processing/combine.py

```diff
@@ -78,6 +36,12 @@ def combine_segments_into_audio_with_timeline(
         shutil.rmtree(segs_folder_path, ignore_errors=True)
         return
 
+    if not timeline:
+        start_ms, _ = parse_audio_filename(str(seg_file_paths[0]))
+        _, end_ms = parse_audio_filename(str(seg_file_paths[-1]))
+        timeline = [(start_ms, end_ms)]
+        max_end_ms = end_ms
+
```

Comment:
When timeline is None, executing this line causes combine_segments_into_audio and combine_segments_into_audio_with_timeline to behave similarly.